ground/src/main/java/com/google/android/ground/persistence/local/stores/LocalSubmissionStore.kt

```diff
@@ -43,16 +43,20 @@ interface LocalSubmissionStore : LocalMutationStore<SubmissionMutation, Submissi
   suspend fun deleteSubmission(submissionId: String)
 
   /**
-   * Emits the list of [SubmissionMutation] instances for a given LOI which match the provided
-   * `allowedStates`. A new list is emitted on each subsequent change.
+   * Returns a [Flow] that emits the list of [SubmissionMutation] instances for a given LOI which
+   * match the provided `allowedStates`. A new list is emitted on each subsequent change.
    */
   fun getSubmissionMutationsByLocationOfInterestIdOnceAndStream(
     survey: Survey,
     locationOfInterestId: String,
     vararg allowedStates: MutationEntitySyncStatus
-  ): Flowable<List<SubmissionMutation>>
+  ): Flow<List<SubmissionMutation>>
```

Comment:
Can we also rename this one while we're at it? getSubmissionMutationsByLoiIdFlow()?

Reply:
Done!